
// public class TryWithMultipleCatchBlock {
//     public static void main(String[] args) {
//         try{

//         }
//         catch(ArithmeticException e)
//         {

//         }
//         catch(Exception e)
//         {

//         }
//     }
// }




// class  Test{
//     public static void main(String[] args) {
//         try{

//         }
//         catch(Exception e)
//         {

//         }
//         catch(Exception e)
//         {

//         }
//     }
// }





// class  Test{
//     public static void main(String[] args) {
//         try{

//         }
//         catch(ArithmeticException e)
//         {

//         }
//         catch(Exception e)
//         {

//         }
//     }
// }








